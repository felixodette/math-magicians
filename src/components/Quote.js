import React from 'react';

const Quote = () => (
  <div id="quote">
    <ul>
      <li>
        &quot;Mathematics is the music of reason.&quot;
        <span>- James Joseph Sylvester</span>
      </li>
      <li>
        &quot;Obvoius is the most dangerous word in Mathematics.&quot;
        <span>- Eric Temple Bell</span>
      </li>
      <li>
        &quot;Mathematics allows for no hypocrisy and no vagueness.&quot;
        <span>- Stendhal</span>
      </li>
      <li>
        &quot;It is impossible to be a Mathematician without being a poet in soul.&quot;
        <span>- Sofia Kovalevskaya</span>
      </li>
      <li>
        &quot;Go down deep enough into anything and you will find Mathematics.&quot;
        <span>- Dean Schlicter</span>
      </li>
    </ul>
  </div>
);

export default Quote;