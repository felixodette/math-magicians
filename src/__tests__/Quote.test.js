import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../components/Quote';
import '@testing-library/jest-dom';

describe('Quote page test', () => {
  it('Quote page render', () => {
    const quotes = renderer
      .create(<Quote />)
      .toJSON();
    expect(quotes).toMatchSnapshot();
  });
});