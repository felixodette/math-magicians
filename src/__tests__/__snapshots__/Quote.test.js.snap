// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote page test Quote page render 1`] = `
<div
  id="quote"
>
  <ul>
    <li>
      "In my school, the brightest boys did math and physics."
      <span>
         - Stephen Hawking
      </span>
    </li>
    <li>
      "Mathematics is written for Mathematicians."
      <span>
        - Nicolaus Corpenicus
      </span>
    </li>
    <li>
      "The truth has never been of any real value to any human being."
      <span>
        - Graham Greene
      </span>
    </li>
    <li>
      "Programming is one of the most difficult branches of applied mathematics."
      <span>
        - Edsger Dijkstra
      </span>
    </li>
    <li>
      "It has been very rare to see a black woman as a protagonist."
      <span>
        - Margot Lee Shetterly
      </span>
    </li>
  </ul>
</div>
`;
